src/arrow_visualizer.jl

```diff
@@ -0,0 +1,50 @@
+struct ArrowVisualizer{V<:AbstractVisualizer}
+    shaft_vis::V
+    head_vis::V
+end
+
+ArrowVisualizer(vis::AbstractVisualizer) = ArrowVisualizer(vis[:shaft], vis[:head])
+
+function setobject!(vis::ArrowVisualizer, material::AbstractMaterial=defaultmaterial();
```

Comment:
I guess this is kind of a non-standard method for this function. What do you think?